packages/tools/examples/dynamicallyAddAnnotations/index.ts

```diff
@@ -240,23 +238,10 @@ const addProgrammaticAnnotation = (
     data: {
       handles: {
         points: [<Types.Point3>[...start], <Types.Point3>[...end]],
-        activeHandleIndex: null,
-        textBox: {
-          hasMoved: false,
-          worldPosition: <Types.Point3>[0, 0, 0],
-          worldBoundingBox: {
-            topLeft: <Types.Point3>[0, 0, 0],
-            topRight: <Types.Point3>[0, 0, 0],
-            bottomLeft: <Types.Point3>[0, 0, 0],
-            bottomRight: <Types.Point3>[0, 0, 0],
-          },
-        },
+        textBox: {},
```

Comment:
do we need textBox: {}? if so we should move it to internal, doesn't makes sense

Reply:
we need it cuz it tries to access textBox.hasMoved